index.js

```diff
@@ -54,14 +54,14 @@ module.exports = class Cutecord extends Plugin {
       ([ msg, channel, n ]) => this.shouldNotify(msg, channel, n)
     )
 
-    const { default: MessageRender } = await getModule(['getElementFromMessageId'])
+    const { default: MessageRender } = await getModule([ 'getElementFromMessageId' ])
     inject(
       'cutecord-messagerender',
       MessageRender,
-      'type', 
+      'type',
       (args) => {
-        const [{ message }] = args
-        if (message.originalMentioned === undefined) {
+        const [ { message } ] = args
+        if (!message.originalMentioned) {
```

Comment:
You should not undo this strict type equality.

Reply:
Didn't catch this auto ESLINT change, reverted back to strict type